tests/App/Models/CollectionTest.php

```diff
@@ -1052,3 +1052,42 @@
         ]);
 
 });
+
+it('sorts collections last time nft was fetched', function () {
+    // fetched yesterday
+    $collection1 = Collection::factory()->create([
+        'extra_attributes' => [
+            'nft_last_fetched_at' => now()->subDays(1),
+        ],
+    ]);
+
+    // fetched one month ago
+    $collection2 = Collection::factory()->create([
+        'extra_attributes' => [
+            'nft_last_fetched_at' => now()->subMonth(),
+        ],
+    ]);
+
+    // never fetched
+    $collection3 = Collection::factory()->create();
+
+    // fetched now
+    $collection4 = Collection::factory()->create([
+        'extra_attributes' => [
+            'nft_last_fetched_at' => now(),
+        ],
+    ]);
+
+    // never fetched
+    $collection5 = Collection::factory()->create();
+
+    $ids = Collection::orderByOldestNftLastFetchedAt()->pluck('id')->toArray();
+
+    expect($ids)->toEqual([
+        $collection3->id,
+        $collection5->id,
```

Comment:
Are you sure that database will always return rows in this order: `collection3` then `collection5`?
Both of these values are `NULL` and we have this `orderBy` logic: `extra_attributes->>\'nft_last_fetched_at\' ASC NULLS FIRST` while  `toEqual` ensures order as well. This test looks like flaky to me..

Reply:
what would you suggest instead for this test

Reply:
Removing `collection5` and having `collection3` would be sufficient as it will prove that `null` value comes before everything else.
I can make the change if you want to proceed.

Reply:
please, then this can get merged